composer.json

```
@@ -21,7 +21,7 @@
     ],
     "require": {
         "ext-json": "*",
-        "php": ">=7.1",
+        "php": ">=7.1||^8.0",
```

Because of consistency with the rest of the `composer.json` file:

```suggestion
        "php": ">=7.1 || ^8.0",
```